bigquery/integration_test.go

```diff
@@ -806,6 +806,12 @@ func TestIntegration_SimpleRowResults(t *testing.T) {
 			query:       "select count(*) from unnest(generate_array(1,1000000)), unnest(generate_array(1, 1000)) as foo",
 			want:        [][]Value{{int64(1000000000)}},
 		},
+		{
+			// Query doesn't yield a result.
+			description: "DML",
+			query:       fmt.Sprintf("CREATE OR REPLACE TABLE %s.%s (foo STRING, bar INT64)", dataset.DatasetID, tableIDs.New()),
+			want:        [][]Value{},
+		},
```

Comment:
Should we run the same test cases with the query preview enabled ?

Reply:
Good point.  I've been doing it out of band but I can swap it.

Reply:
And testing was a wise recommendation, helped me discover an issue where the stateless result is empty.  Previously we'd fallback, but there's no fallback available.